### PR TITLE
feat: skip java db

### PR DESCRIPTION
**Pull request by chen-keinan**
## Description
 Add support for `--skip-java-update` parameter to skip java db update 

## Related
- https://github.com/goharbor/harbor/issues/19759

### CODE REVIEW

File: helm/harbor-scanner-trivy/values.yaml [hunk 1]
```diff
@@ -74,6 +74,10 @@ scanner:
     ## If the flag is enabled you have to manually download the `trivy.db` file and mount it in the
     ## `/home/scanner/.cache/trivy/db/trivy.db` path (see `cacheDir`).
     skipUpdate: false
+       
+    ## If the flag is enabled you have to manually download the `trivy-java-db` file and mount it in the
```
Comment (knqyf263) [inline helm/harbor-scanner-trivy/values.yaml:78]: nit:
```suggestion
    ## If the flag is enabled you have to manually download the `trivy-java.db` file and mount it in the
```

File: helm/harbor-scanner-trivy/values.yaml [hunk 2]
```diff
@@ -74,6 +74,10 @@ scanner:
...
+    ## `/home/scanner/.cache/trivy/db/trivy.java-db` path (see `cacheDir`).
```
Comment (knqyf263) [inline helm/harbor-scanner-trivy/values.yaml:79]: Is it different from the default path?
```suggestion
    ## `/home/scanner/.cache/trivy/java-db/trivy-java.db` path (see `cacheDir`).
```

File: pkg/http/api/v1/handler_test.go [hunk 3]
```diff
@@ -483,6 +484,7 @@ func TestRequestHandler_GetMetadata(t *testing.T) {
       "org.label-schema.vcs-ref": "abc",
       "org.label-schema.version": "0.1",
       "env.SCANNER_TRIVY_SKIP_UPDATE": "false",
+	  "env.SCANNER_TRIVY_SKIP_JAVA_DB_UPDATE": "false",
```
Comment (knqyf263) [inline pkg/http/api/v1/handler_test.go:487]: nit: The indent seems to be broken

File: pkg/http/api/v1/handler_test.go [hunk 4]
```diff
@@ -535,6 +538,7 @@ func TestRequestHandler_GetMetadata(t *testing.T) {
       "org.label-schema.vcs-ref": "abc",
       "org.label-schema.version": "0.1",
       "env.SCANNER_TRIVY_SKIP_UPDATE": "false",
+	  "env.SCANNER_TRIVY_SKIP_JAVA_DB_UPDATE":"false",
```
Comment (knqyf263) [inline pkg/http/api/v1/handler_test.go:541]: same here

File: pkg/http/api/v1/handler_test.go [hunk 5]
```diff
@@ -584,6 +588,7 @@ func TestRequestHandler_GetMetadata(t *testing.T) {
...
```
Comment (knqyf263) [inline pkg/http/api/v1/handler_test.go:591]: ditto

File: test/integration/api/rest_api_test.go [hunk 6]
```diff
@@ -213,6 +214,7 @@ func TestRestApi(t *testing.T) {
     "org.label-schema.vcs-ref": "abc",
     "org.label-schema.vcs": "https://github.com/aquasecurity/harbor-scanner-trivy",
     "env.SCANNER_TRIVY_SKIP_UPDATE": "false",
+	"env.SCANNER_TRIVY_SKIP_JAVA_DB_UPDATE": "false",
```
Comment (knqyf263) [inline test/integration/api/rest_api_test.go:217]: ditto

**Review by knqyf263 — COMMENTED** [review]
I left small comments

**Comment by chen-keinan** [comment]
@knqyf263 comments has been addressed

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
@chen-keinan LGTM! You can merge this PR anytime.

**Pull request merged**